organizations/user.go

```diff
@@ -171,3 +171,41 @@ func UpdateProfilePicture(w http.ResponseWriter, r *http.Request) {
 	utils.GetSuccess("Profile picture updated", result, w)
 
 }
+
+// Delete single member from an organizatin
+func DeleteMember(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	member_collection, org_collection := "members", "organizations"
+	orgId := mux.Vars(r)["id"]
+	memberId := mux.Vars(r)["mem_id"]
+
+	pOrgId, err := primitive.ObjectIDFromHex(orgId)
+	if err != nil {
+		utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+		return
+	}
+
+	// get organization
+	orgDoc, _ := utils.GetMongoDbDoc(org_collection, bson.M{"_id": pOrgId})
+	if orgDoc == nil {
+		fmt.Printf("org with id %s doesn't exist!", orgId)
+		utils.GetError(errors.New("operation failed"), http.StatusBadRequest, w)
+		return
+	}
+
+	query, _ := primitive.ObjectIDFromHex(memberId)
+	checkMember, err := utils.GetMongoDbDoc(member_collection, bson.M{"_id": query})
+	if checkMember == nil {
+		fmt.Printf("Member with ID: %s does not exists ", memberId)
+		utils.GetError(err, http.StatusBadRequest, w)
+		return
+	}
+
+	delOrgMember, err := utils.DeleteOneMongoDoc(member_collection, memberId)
+	if err != nil {
+		utils.GetError(err, http.StatusInternalServerError, w)
+		return
+	}
+	utils.GetSuccess("Successfully Deleted Member", delOrgMember.DeletedCount, w)
```

Comment:
lets do something like this instead of returning a number that the user might not need or understand

if response.DeletedCount == 0 {
		utils.GetError(errors.New("operation failed"), http.StatusInternalServerError, w)
		return
	}

	utils.GetSuccess("organization deleted successfully", nil, w)